content/projects/jam/index.adoc

```diff
@@ -10,49 +10,46 @@ links:
+* Support specific to JAMs - As CDF will manage the operational aspect of JAMs and CI/CD meetups, the Advocacy & OutReach SIG is a channel to provide guidance/advice to new JAM organizers if needed. Email jenkins-advocacy-and-outreach-sig@googlegroups.com 
 
 === Best Practices
 
-* Attend existing meetup groups to gauge Jenkins interest/market within your area.
-* Reach out to existing JAM organizer(s) to learn best practices and time commitment.
-* Set up a Twitter account (for example, #JenkinsNYC). Email your group’s Twitter ID to jenkinsci-jam@googlegroups.com so we can help to promote your JAM.
-* Promote, tweet and post your JAM in advance.
+* Attend existing meetup groups to gauge any of CDF's current projects interest/market within your area.
+* Reach out to existing CDF/JAM organizer(s) to learn best practices and time commitment.
+* Set up a Twitter account (for example, #JenkinsNYC). Email your group’s Twitter ID to meetups@cd.foundation so we can help to promote your meetup.
+* Promote, tweet and post your meetup in advance.
 * Avoid product pitches. Keep it real, it’s not about promotions. This is about - and for - learning opportunities for the community.
 * Shake up the format. Have presentations, panel discussions, roundtables, workshops, hackathons, etc.
-* Set a consistent date, time and location so members will plan for it. Avoid cancelling your meeting as people will lose faith and stop RSVPing for your JAM.
+* Set a consistent date, time and location so members will plan for it. Avoid cancelling your meeting as people will lose faith and stop RSVPing for your meetup.
+* Review the CDF meetups repository on Github https://github.com/cdfoundation/meetups
 
 === Stepping Down
 
-In the event you can no longer be a JAM organizer, we ask that you nominate a
+In the event you can no longer be an organizer, we ask that you nominate a
 replacement organizer in your place. If this is not possible please send email
-to jenkinsci-jam@googlegroups.com, we will work with you to find a replacement.
+to meetups@cd.foundation, we will work with you to find a replacement.
 
-== Supporting a JAM
+== Supporting a meetup group
 
-If you are not able to be a JAM organizer, speakers, food or venue, or recording sponsors are always needed. If your company is interested in sponsoring any of these items pls email jenkinsci-jam@googlegroups.com or reach out to the local JAM organizer via meetup.com or fill out link:https://docs.google.com/a/cloudbees.com/forms/d/1dGpwxpwoJDHR3fTlIcFXO8GZVpx5i_dWUlbi9LKolX4/edit[this form]. 
+If you are not able to be an organizer, speakers, food or venue, or recording sponsors are always needed. If your company is interested in sponsoring any of these items pls email meetups@cd.foundation or reach out to the local organizer via meetup.com or fill out link:https://docs.google.com/a/cloudbees.com/forms/d/1dGpwxpwoJDHR3fTlIcFXO8GZVpx5i_dWUlbi9LKolX4/edit[this form]. 
```

Comment:
Is this CloudBees form still maintainer @alyssat ? I would recommend either removing it entirely or switching to a Google form which would be accessible to the Advocacy&Outreach SIG